x/evm/keeper/state_transition_benchmark_test.go

```diff
@@ -0,0 +1,134 @@
+package keeper_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/stretchr/testify/require"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+)
+
+var templateTx = &ethtypes.LegacyTx{
+	GasPrice: big.NewInt(1),
+	Gas:      21000,
+	To:       &common.Address{},
+	Value:    big.NewInt(0),
+	Data:     []byte{},
+}
+
+func newSignedEthTx(
+	txData *ethtypes.LegacyTx,
```

Comment:
this should use the tx data interface from geth

Reply:
fixed!